src/logic/create/provider/security.ts

```diff
@@ -30,35 +38,41 @@ export async function selectSecurity(
 async function enterSecuritySchema(
   baseUrl: string
 ): Promise<SecurityScheme | 'none'> {
-  const schema: 'api key token' | 'bearer token' | 'basic' | 'digest' | 'none' =
-    (
-      await inquirer.prompt<{
-        schema: 'api key token' | 'bearer token' | 'basic' | 'digest' | 'none';
-      }>({
-        name: 'schema',
-        message: `Select authentication method for ${baseUrl}:`,
-        type: 'list',
-        choices: ['API Key', 'Bearer Token', 'Basic Auth', 'Digest', 'none'],
-      })
-    ).schema;
+  const schema: SecuritySchemeName = (
+    await inquirer.prompt<{
+      schema: SecuritySchemeName;
+    }>({
+      name: 'schema',
+      message: `Select authentication method for ${baseUrl}:`,
+      type: 'list',
+      choices: [
+        'API Key',
+        'Bearer Token',
+        'Basic Authentication',
+        'Digest Authentication',
+        'none',
+      ],
```

Comment:
Wouldn't this programmatic resolve be even safer?

```suggestion
      choices: Object.values(SecuritySchemeName),
```